### PR TITLE
[T5A1][F09-B4] Tan Yong Sheng

### PR DESCRIPTION
**Pull request by e-tys**

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -43,9 +43,12 @@ Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
```
Comment (chanjunweimy) [inline doc/DeveloperGuide.md]: when the user** :)

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -43,9 +43,12 @@ Priority | As a ... | I want to ... | So that I can...
...
 `* * *` | user | add a new person |
```
Comment (chanjunweimy) [inline doc/DeveloperGuide.md]: what is the benefit of adding a new person?

File: doc/DeveloperGuide.md [hunk 3]
```diff
@@ -43,9 +43,12 @@ Priority | As a ... | I want to ... | So that I can...
...
+`* * *` | user | update a person details | keep the information up to date
```
Comment (chanjunweimy) [inline doc/DeveloperGuide.md]: keep what information up to date?

File: doc/DeveloperGuide.md [hunk 4]
```diff
@@ -43,9 +43,12 @@ Priority | As a ... | I want to ... | So that I can...
...
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+`* * *` | user | find a person by phone number | locate details of persons without having to go through the entire list
```
Comment (chanjunweimy) [inline doc/DeveloperGuide.md]: good job :)

File: doc/DeveloperGuide.md [hunk 5]
```diff
@@ -74,12 +77,36 @@ Use case ends.
...
 
+#### Use case: Renaming existing tag
+
+**MSS**
+
+1. User requests to rename a tag
+2. AddressBook ask for a confirmation for the renaming request
+3. User confirms the request
+4. AddressBook rename the tag to new name
+Use case ends.
```
Comment (chanjunweimy) [inline doc/DeveloperGuide.md]: good job :)

File: doc/DeveloperGuide.md [hunk 6]
```diff
@@ -74,12 +77,36 @@ Use case ends.
...
+> Use case ends
+
 ## Appendix C : Non Functional Requirements
 
 1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java 8 or higher installed.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. GUI should user-friendly and intuitive.
+6. Should have quick response time.
```
Comment (chanjunweimy) [inline doc/DeveloperGuide.md]: what is "quick"?

File: doc/DeveloperGuide.md [hunk 7]
```diff
@@ -74,12 +77,36 @@ Use case ends.
...
+
+> Use case ends
+
 ## Appendix C : Non Functional Requirements
 
 1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java 8 or higher installed.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. GUI should user-friendly and intuitive.
```
Comment (chanjunweimy) [inline doc/DeveloperGuide.md]: what is user-friendly?
what is intuitive?

File: doc/DeveloperGuide.md (hunk 3 above)
Comment (e-tys) [inline doc/DeveloperGuide.md]: Personal Information such as address. phone number, email.

File: doc/DeveloperGuide.md (hunk 7 above)
Comment (e-tys) [inline doc/DeveloperGuide.md]: What I meant about user-friendly is about the number to steps required for the user to get his job done.
Intuitive = no lag in loading of the interface among mouse clicks or execution of command

File: doc/DeveloperGuide.md (hunk 6 above)
Comment (e-tys) [inline doc/DeveloperGuide.md]: A response time of 0.5 ms?

**Pull request closed without merging**